### PR TITLE
DAT-98765-Multi-select-focus-on-reset

### PR DESCRIPTION
**Pull request by itaydafna**
## Short Description

Don't focus inline-search on an external controlled change.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Are you adding a new package?
- [ ] yes
- [X] no

## PR Checklist
- [X] I'm using the theme colors from styled-components and checked my component in dark mode as well
- [X] Add types (Prop-types / typescript)
- [X] Commit your changes
- [X] Run `npm version **` [minor, major, patch]. Commit changes
- [X] Run `npm run lib:build`. Commit changes
- [X] Run `npm run lib:version`. Commit changes
- [X] Push changes and open a pull request

### CODE REVIEW

File: app/src/components/base/Select/InlineSearch.js [hunk 1]
```diff
@@ -22,11 +22,15 @@ class InlineSearch extends Component {
     onSelect: PropTypes.func,
     toggleFocus: PropTypes.func,
     onKeyDown: PropTypes.func,
-    onKeyUp: PropTypes.func
+    onKeyUp: PropTypes.func,
+    isLastValueChangeTriggeredLocally: PropTypes.bool
```
Comment (itaydafna) [inline app/src/components/base/Select/InlineSearch.js:26]: Do you have a better name for this? I tried to make it as clear as possible.

File: app/src/components/base/Select/InlineSearch.js [hunk 2]
```diff
@@ -57,6 +61,11 @@ class InlineSearch extends Component {
     this.props.onSearch(el);
   };
 
+  deleteTag = option => {
+    this.setState({ isLastValueChangeTriggeredLocally: true });
```
Comment (Chikel) [inline app/src/components/base/Select/InlineSearch.js:65]: Is this state field used anywhere?

File: app/src/components/base/Select/InlineSearch.js (hunk 1 above)
Comment (Chikel) [reply app/src/components/base/Select/InlineSearch.js:26]: I think it's accurate, let's keep it.

**Review by Chikel — CHANGES_REQUESTED** [review]

File: app/src/components/base/Select/InlineSearch.js (hunk 2 above)
Comment (itaydafna) [reply app/src/components/base/Select/InlineSearch.js]: No, it isn't, redundant code. Nice catch!

**Review by Chikel — APPROVED** [review]

**Pull request closed without merging**